### DISCUSSION TITLE
Shinou's Discussions Place.

### DISCUSSION THREAD

**Discussion by YoruAkio** [YoruAkio/Shinou#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by YoruAkio** [comment]
Whats up Brudda.

**Comment by AkioYoru** [comment]
Hello Guys
